### PR TITLE
valid factorial number update under the tests

### PR DESCRIPTION
**Pull request by ghost**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ tests:
     testString: assert(typeof factorial(2) === 'number');
   - text: <code>factorial(3)</code> should return 6.
     testString: assert.equal(factorial(3),results[0]);
-  - text: <code>factorial(3)</code> should return 120.
+  - text: <code>factorial(5)</code> should return 120.
     testString: assert.equal(factorial(5),results[1]);
-  - text: <code>factorial(3)</code> should return 3,628,800.
+  - text: <code>factorial(10)</code> should return 3,628,800.
     testString: assert.equal(factorial(10),results[2]);
```
Comment (thecodingaviator) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md]:
```suggestion
    testString: assert.equal(factorial(10), 3628800);
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md [hunk 2]
```diff
@@ -36,9 +36,9 @@ tests:
     testString: assert(typeof factorial(2) === 'number');
   - text: <code>factorial(3)</code> should return 6.
     testString: assert.equal(factorial(3),results[0]);
-  - text: <code>factorial(3)</code> should return 120.
+  - text: <code>factorial(5)</code> should return 120.
     testString: assert.equal(factorial(5),results[1]);
```
Comment (thecodingaviator) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md]:
```suggestion
    testString: assert.equal(factorial(5), 120);
```

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md [hunk 3]
```diff
@@ -36,9 +36,9 @@ tests:
     testString: assert(typeof factorial(2) === 'number');
   - text: <code>factorial(3)</code> should return 6.
     testString: assert.equal(factorial(3),results[0]);
```
Comment (thecodingaviator) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/factorial.english.md]:
```suggestion
    testString: assert.equal(factorial(3), 6);
```

**Review by thecodingaviator — CHANGES_REQUESTED** [review]
Please also remove After Test portion section. I don't see why we're doing it like this in a convoluted way when we can outright check the results

cc: @ojeytonwilliams what do you think?

**Comment by thecodingaviator** [comment]
@sybiljasmine please wait till one of the other mods reply too. In case we don't want to do this, I don't want to waste your time

**Comment by ojeytonwilliams** [comment]
I agree with @thecodingaviator, it's clearer to have the individual values in the assertions.  @sybiljasmine could you remove the `results` array entirely?

**Review by thecodingaviator — APPROVED** [review]

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Congrats @sybiljasmine on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**